cleverhans/attacks_tf.py

```diff
@@ -21,12 +21,15 @@ def fgsm(x, predictions, eps=0.3, clip_min=None, clip_max=None):
                clip_max=clip_max)
 
 
-def fgm(x, preds, y=None, eps=0.3, ord=np.inf, clip_min=None, clip_max=None):
+def fgm(x, preds, y=None, eps=0.3, ord=np.inf,
+        clip_min=None, clip_max=None,
+        targeted=False):
     """
     TensorFlow implementation of the Fast Gradient Method.
     :param x: the input placeholder
     :param preds: the model's output tensor
-    :param y: (optional) A placeholder for the model labels. Only provide
+    :param y: (optional) A placeholder for the model labels. If targeted
```

Comment:
Well I guess it's for both untargeted and targeted, so it's fine to leave it like that